packages/eslint-plugin/src/rules/prefer-regexp-exec.ts

```diff
@@ -72,22 +72,24 @@ export default createRule({
       return result;
     }
 
-    function isLikelyToContainGlobalFlag(
+    function provablyDoesNotContainGlobalFlag(
```

Comment:
I read this as "probably" on first sight, and was like "okay so it's still guesswork?" 😅

Maybe "definitely" is better, or not have the adverb at all? IDK

Reply:
Reasonable. Changed it up a bit, hopefully that is a little more clear at a glance 👍